langlib/langlib-test/src/test/resources/test-src/regexp_test.bal

```diff
@@ -1311,6 +1311,17 @@ function testTranslatingDiffNodesInCharClass() {
     assertEquality("A", (<regexp:Span>res8).substring());
 }
 
+function testRegexpWithUnicodeChars() {
+    regexp:Span? resA1 = regexp:find(re = re `\u{1F30D}`, str = "😀 Hello, 🌍! ❤️");
```

Comment:
no need to call as named arguments unless you really need. Add one such case and change others to positional arguments

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/40368/commits/e26daea0c1d5b4a0c11d068f104eda74dd5ce5c5.